src/content/security/cluster-access/index.md

```diff
@@ -19,6 +19,8 @@ user_questions:
   - How do I get access my clusters?
   - Who has access to my clusters?
   - Why do Giant Swarm personnel need access to my clusters?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-ludacris
```

Comment:
FYI @giantswarm/team-ludacris - assigning a security related page to you.

Reply:
:+1:

Reply:
Not blocking here but what is the decision between putting a doc into sig-docs compared to a team?

Reply:
SIG docs owns the glue pages (section pages) that carry almost no content, or that would be hard to assign otherwise. But if you feel like a team should own one that is assigned to SIG docs, let me know.